modules/openxr/doc_classes/OpenXRExtensionWrapper.xml

```diff
@@ -15,13 +20,15 @@
 			<description>
 				Returns a pointer to an [code]XrCompositionLayerBaseHeader[/code] struct to provide the given composition layer.
 				This will only be called if the extension previously registered itself with [method OpenXRAPIExtension.register_composition_layer_provider].
+				[b]Note:[/b] This virtual method will be called on the render thread. Additionally, the data it returns will be used shortly after this method is called, so it needs to remain valid until then.
```

Comment:
The "until then" is a bit vague here, when read literally it seems to imply the data must be valid until the method is called. I think this needs a clarification of how to ensure the data is correctly handled, some suggestion on how to make sure you don't invalidate it too early